app/react/components/PageHeader/ContextHelp/ContextHelp.tsx

```diff
@@ -17,7 +17,6 @@ export function ContextHelp() {
       <div
         className={clsx(
           headerStyles.menuIcon,
-          'menu-icon',
```

Comment:
Could be worth removing .menu-icon::after from ContextHelp.css too

Reply:
Removing menu icon seems like it stops applying the `cursor: pointer;` style from `.menu-icon` in ContextHelp.css.

It could be worth keeping the class 'menu-icon' and just removing `.menu-icon::after` from `ContextHelp.css`